### ISSUE TITLE
Opacity no longer works in MSIE8 [regression]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#235]
```
http://ie7-js.googlecode.com/svn/test/opacity.html

```

Original issue reported on code.google.com by `dean.edw...@gmail.com` on 18 Mar 2010 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
they changed it
here's how to do it for 90%
-ms-filter: "progid:DXImageTransform.Microsoft.Alpha(opacity=90)";
```

Original comment by `fun...@gmail.com` on 21 Mar 2010 at 1:43

**Comment by GoogleCodeExporter** [comment]
```
you can also use:

filter:alpha(opacity=90);
```

Original comment by `florian....@gmail.com` on 22 Mar 2010 at 9:05

**Comment by GoogleCodeExporter** [comment]
```
no
read 
http://blogs.msdn.com/ie/archive/2008/09/08/microsoft-css-vendor-extensions.aspx
```

Original comment by `fun...@gmail.com` on 23 Mar 2010 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
Somehow even with the 2.0(beta3) branches I can't get opacity to work anymore. 
Is
there currently any usable workaround?

If not I'd definitely suggest upping priority from medium since many sites can
fundamentally break on missing opacity support (consider mandatory data missing 
from
view).
```

Original comment by `ni...@keurentjes.nl` on 30 Mar 2010 at 11:10

**Comment by GoogleCodeExporter** [comment]
```
I have managed to fix this bug. It will be in the next release of IE7.js.
```

Original comment by `dean.edw...@gmail.com` on 31 Mar 2010 at 9:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 25 Apr 2010 at 2:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1 beta4.
```

Original comment by `dean.edw...@gmail.com` on 30 Apr 2010 at 9:14
- Changed state: **Fixed**
